DiffPriv/enc.py

```diff
@@ -215,3 +222,179 @@ def decrypt(self, text: str, key: str):
                 decrypted.append(brokentext[x])
 
         return ''.join(decrypted)
+
+class SSS(object):
+    """
+    ## Shamir's Secret Sharing System (abbreviated SSS)
+    Wikipedia: https://en.wikipedia.org/wiki/Shamir's_Secret_Sharing
+
+    ---
+
+    Shamir's Secret Sharing System provides a way of dividing up a secret,
+    S, among N parties. S can *only* be found if all N parties put their keys together.
+    Shamir's Secret Sharing System uses the coefficients of a polynomial to encode the data
+    and assigns each party a point on the polynomial.
+    """
+    def __init__(self, bytes):
+        """Initializes Shamir's Secret Sharing System with data from parameter `bytes`"""
+        self.data = str(bytes)
+        self.bytes = len(self.data)
+        self.shares = []
+
+    def divide(self, n, quiet=False):
+        """
+        Returns a tuple of digits from the data, dividing it up into `n` shares
+        Turn on quiet mode with `quiet=True`
+        """
+        self.data = str(self.data)
+        if n > self.bytes:
+            n = self.bytes
```

Comment:
We need testing for lines 251-252 and 263-266 of `enc.py`. Please add the tests and we can move!

Reply:
@quantum9Innovation when is this going to be finished?

Reply:
Now

Reply:
Fixed with `# pragma: no cover`